packages/core/src/routes/connector.ts

```diff
@@ -122,11 +122,28 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
         })
       );
 
+      const insertConnectorId = generateConnectorId();
       ctx.body = await insertConnector({
-        id: generateConnectorId(),
+        id: insertConnectorId,
         ...body,
       });
 
+      if (
+        connectorFactory.type === ConnectorType.Sms ||
+        connectorFactory.type === ConnectorType.Email
+      ) {
+        const logtoConnectors = await getLogtoConnectors();
```

Comment:
add comment on that?

and thoes options can be achieved by a "delete where" query?

Reply:
Seems we can't, we do not store connector `type` in DB.